src/main/java/bio/terra/service/snapshotbuilder/query/QueryBuilder.java

```diff
@@ -0,0 +1,48 @@
+package bio.terra.service.snapshotbuilder.query;
+
+import java.util.List;
+
+public class QueryBuilder {
```

Comment:
Can you make this a static inner class of `Query`, called `Builder`? That is how I'd expect a builder pattern for a class to be written.

Reply:
Yes, this was my first implementation but I took the class out for simplicity purposes.